types/react-typing-animation/index.d.ts

```diff
@@ -0,0 +1,26 @@
+// Type definitions for react-typing-animation 1.6
+// Project: https://adamjking3.github.io/react-typing-animation-example/
```

Comment:
can you amend this to working url? Like `https://github.com/notadamking/react-typing-animation#readme`
Thanks!